runHarness/DataManagers/AuctionKubernetesDataManager.pm

```diff
@@ -166,8 +160,18 @@ sub prepareDataServices {
 		$appInstance->clearDataServicesBeforeStart($logPath);
 	}
 	my $allIsStarted = $appInstance->startServices("data", $logPath, 0);
+	
+	if (!$allIsStarted) {
+		# Delete the PVCs for this namespace and try again
+		$console_logger->info(
+			"Couldn't start data services for appInstance $appInstanceNum of workload $workloadNum. Clearing data and retrying.\n" );
+		$appInstance->stopServices("data", $logPath);
+		my $cluster = $self->host;
+		my $namespace = $self->appInstance->namespace;
+		$cluster->kubernetesDeleteAllWithLabelAndResourceType("app=auction", "pvc", $namespace );
+		$allIsStarted = $appInstance->startServices("data", $logPath, 0);	
+	}
```

Comment:
Should there be an option to turn this auto deleting off? Should it be default?
Maybe data services didn't start for some other reason and its a medium+ config size that takes a while to reload.

Reply:
This makes sense.  I will add the option and have it off (no auto-delete) by default.  I will also update the documentation.

Reply:
If it's off by default, @hrosenbe could you add a message to "Couldn't bring to running all data services for appInstance 12 of workload 1." that "If this problem recurs, you can enable auto-remediation by ..."?

Reply:
Good idea.  I will add this also.